lib/UnoCore/Targets/iOS/@(Project.Name)/@(Project.Name).entitlements

```diff
@@ -3,12 +3,14 @@
 <plist version="1.0">
 <dict>
 
+#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
```

Comment:
`__IPHONE_13_0` and `__IPHONE_OS_VERSION_MAX_ALLOWED` are C preprocessor macros and can't be used inside XML files.

Reply:
darn, its for the case of me handing you or a fellow dev the same project with the entitlement set but you don't have access to the entitlement, so it breaks when you try to build

Reply:
Oh... That sounds like an iffy situation, but then plain native Xcode projects must have the same issue as well? How is this usually solved in Xcode projects?

With Uno, it should be possible to explicitly disable the feature from the command line, though, like so: 
```
uno build ios --set:Project.iOS.SystemCapabilities.SignInWithApple=0
```

Or, the other way around, explicitly enabling the feature when it's needed:
```
uno build ios --set:Project.iOS.SystemCapabilities.SignInWithApple=1
```

Reply:
Yeah, unfortunately the way forward is to literally buy a new Mac, upgrade to new OS and Xcode lol, thanks @mortend, will give it a try

Reply:
Yeah, that's Apple Style. Cheers @AndrewEQ!

Reply:
I can confirm that `uno build ios --set:Project.iOS.SystemCapabilities.SignInWithApple=0` works! 😄 🎉  So now we just gotta remember to write about this somewhere 😆